storage/statedb/cache_fastcache.go

```diff
@@ -52,30 +62,64 @@ func NewFastCache(config *TrieNodeCacheConfig) TrieNodeCache {
 		return nil
 	}
 
-	fc := &FastCache{fast: fastcache.LoadFromFileOrNew(config.FastCacheFileDir, config.LocalCacheSizeMB*1024*1024)} // Convert MB to Byte
-	stats := fc.UpdateStats().(fastcache.Stats)
+	fc := &FastCache{new: fastcache.New(config.LocalCacheSizeMB * 1024 * 1024)} // Convert MB to Byte
+	go fc.loadFromBackground(config)
 
-	logger.Info("Initialize local trie node cache (fastCache)",
-		"MaxMB", config.LocalCacheSizeMB, "FilePath", config.FastCacheFileDir,
-		"LoadedBytes", stats.BytesSize, "LoadedEntries", stats.EntriesCount)
+	fmt.Println("Initialize local trie node cache (fastCache)",
+		"MaxMB", config.LocalCacheSizeMB, "FilePath", config.FastCacheFileDir)
 	return fc
 }
 
+func (cache *FastCache) loadFromBackground(config *TrieNodeCacheConfig) {
+	start := time.Now()
+	cache.old = fastcache.LoadFromFileOrNew(config.FastCacheFileDir, config.LocalCacheSizeMB*1024*1024)
```

Comment:
The caches use too much memory and there could be an OOM during upload. What do you think of resizing the cache?

Reply:
@winnie-byun If we resize the cache, we can't load the stored fastcache since it checks if the current configuration is identical with the previous, stored one.